trail/trail/urls.py

```diff
@@ -29,6 +45,7 @@
     # in the new template the index is the gallery, we don't need so many apps
     re_path(r'^index/', include('gallery.urls')),
     re_path(r'^',  include('gallery.urls')),
+    path('swagger', schema_view.with_ui(cache_timeout=0), name="swagger-json"),
```

Comment:
Probably better to name the link something like "api_docs" and add a link somewhere on the query page (open issue or trello ticket).